spring-grpc-spring-boot-autoconfigure/src/test/java/org/springframework/grpc/autoconfigure/server/health/GrpcServerHealthAutoConfigurationTests.java

```diff
@@ -55,7 +55,8 @@ class GrpcServerHealthAutoConfigurationTests {
 
 	private ApplicationContextRunner contextRunner() {
 		return new ApplicationContextRunner()
-			.withConfiguration(AutoConfigurations.of(GrpcServerHealthAutoConfiguration.class));
```

Comment:
I think the proper test to add here should answer the following: 

"The auto-config now relies on a bindable service bean to be defined or the auto-config is skipped". This is the simplest test to do that IMO:

```java
@Test
void whenNoBindableServiceDefinedDoesNotAutoConfigureBean() {
new ApplicationContextRunner()
		.withConfiguration(AutoConfigurations.of(GrpcServerHealthAutoConfiguration.class))
		.run((context) -> assertThat(context).doesNotHaveBean(GrpcServerHealthAutoConfiguration.class));
}


```